modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/chart/bar/MultiBarChart.es.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import React, {useState} from 'react';
+import {
+	Bar,
+	BarChart,
+	Cell,
+	Legend,
+	ResponsiveContainer,
+	Tooltip,
+	XAxis,
+	YAxis,
+} from 'recharts';
+
+import colors, {NAMED_COLORS} from '../../../utils/colors.es';
+import TooltipContent from '../TooltipContent.es';
+import CustomizedAxisTick from './CustomizedAxisTick.es';
+
+const {blueDark, gray} = NAMED_COLORS;
+
+export default ({data, field, height, structure, width}) => {
+	const [activeIndex, setActiveIndex] = useState(null);
+
+	const {columns} = structure;
+
+	const handleOnMouseOut = () => {
+		setActiveIndex(null);
+	};
+
+	const handleOnMouseOver = (index) => {
+		setActiveIndex(index);
+	};
+
+	const getColumnLabel = (column) => {
+		return field.columns[column];
+	};
+
+	const getRowLabel = (row) => {
+		return field.rows[row];
+	};
+
+	const processData = ({columns, rows}) => {
+		const newData = [];
+
+		rows.map((row) => {
+			const newDataAux = {name: getRowLabel(row)};
+
+			columns.map((col) => {
+				newDataAux[getColumnLabel(col)] = data[row][col]
+					? data[row][col]
+					: 0;
+			});
+			newData.push(newDataAux);
+		});
+
+		return newData;
+	};
+
+	data = structure ? processData(structure) : data;
+
+	let xAxisProps = {};
+	let yAxisProps = {};
+
+	xAxisProps = {
+		['axisLine']: {stroke: blueDark},
+		['dataKey']: 'name',
+		['interval']: 0,
+	};
+
+	yAxisProps = {
+		['axisLine']: {stroke: gray},
+		['type']: 'number',
+	};
```

Comment:
I forgot to remove it. Initially, I was making code generic to reuse the variables because I was trying to unify the code of SimpleBarChart with MultiBarChar, but it didn't work since there're many specific things.